File: sdkwork-app-sdk-go/types/popup_advert_vo.go
package types

// 弹窗广告VO
type PopupAdvertVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	AdvertId string `json:"advertId"`
	AdvertName string `json:"advertName"`
	AdvertType string `json:"advertType"`
	PositionId string `json:"positionId"`
	CreativeUrl string `json:"creativeUrl"`
	LandingUrl string `json:"landingUrl"`
	Priority int `json:"priority"`
	Enabled bool `json:"enabled"`
	StartTime string `json:"startTime"`
	EndTime string `json:"endTime"`
	ClickCount int `json:"clickCount"`
	ImpressionCount int `json:"impressionCount"`
	PopupSize string `json:"popupSize"`
	DisplayDuration int `json:"displayDuration"`
	CloseButtonEnabled bool `json:"closeButtonEnabled"`
	PopupPosition string `json:"popupPosition"`
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/PopupAdvertVO.kt
package com.sdkwork.app

data class PopupAdvertVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val advertId: String? = null,
    val advertName: String? = null,
    val advertType: String? = null,
    val positionId: String? = null,
    val creativeUrl: String? = null,
    val landingUrl: String? = null,
    val priority: Int? = null,
    val enabled: Boolean? = null,
    val startTime: String? = null,
    val endTime: String? = null,
    val clickCount: Int? = null,
    val impressionCount: Int? = null,
    val popupSize: String? = null,
    val displayDuration: Int? = null,
    val closeButtonEnabled: Boolean? = null,
    val popupPosition: String? = null
)
